app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -44,7 +44,15 @@ function ServiceInstanceRow($filter, DataService, rowMethods, $uibModal) {
     row.notifications = rowMethods.getNotifications(row.apiObject, row.state);
     row.displayName = getDisplayName();
     row.description = getDescription();
-    row.instanceBindings = getBindings();
+  };
+
+  var prevBindings = [];
+  row.$doCheck = function() {
+    var bindings = getBindings();
+    if(prevBindings.length !== bindings.length) {
```

Comment:
Instead of having to filter `row.state.bindings` every digest loop, can we keep a map of bindings by instance ref name updated in the overview watch callback? Then `$doCheck` might not be necessary.